test/Dialect/FIRRTL/omir.mlir

```diff
@@ -0,0 +1,286 @@
+// RUN: circt-opt -firrtl-lower-annotations -split-input-file %s | FileCheck %s
+
```

Comment:
This file is a conversion of `omir.fir` to MLIR.

Reply:
It's great that we can have these on MLIR now, so we're not testing the FIRRTL parser all the time alongside the anno scattering :grimacing:

Reply:
It makes it a lot cleaner. One sharp edge is that `firtool -parse-only` now usually requires `| circt-opt -firrtl-lower-annotations`. For usability, it may be reasonable to have `-parse-only` also add the `LowerAnnotations` pass.

Reply:
Yeah I think that makes a lot of sense. You can still disable the anno lowering explicitly if you don't want it. But I agree that in 99% of cases you'll want the lowered annos anyway.